packages/aws-cdk-lib/aws-ecs-patterns/lib/base/application-load-balanced-service-base.ts

```diff
@@ -391,6 +391,15 @@ export interface ApplicationLoadBalancedTaskImageOptions {
   * @default none
   */
   readonly command?: string[];
+
+  /**
+   * The amount (in GiB) of ephemeral storage to be allocated to the task. The maximum supported value is 200 GiB.
+   *
+   * NOTE: This parameter is only supported for tasks hosted on AWS Fargate using platform version 1.4.0 or later.
+   *
+   * @default - Undefined, in which case, the task will receive 20GiB ephemeral storage.
```

Comment:
```suggestion
   * @default - Undefined, in which case, the task will receive 20 GiB ephemeral storage.
```
smae for the other files

Reply:
I mentioned this as per aws-ecs package. There is no space between 20 and GiB. I
But I made changes for 200 GiB -> 200GiB which I mentioned in the same description, to make it consistent as @iamhopaul123 suggested me in the slack.